packages/mds-agency/request-handlers.ts

```diff
@@ -83,24 +106,24 @@ export const registerVehicle = async (req: AgencyApiRequest, res: AgencyApiRespo
     } catch (err) {
       logger.error('writeRegisterEvent failure', err)
     }
-    res.status(201).send({ result: 'register device success', recorded, device })
+    const { version } = res.locals
+    res.status(201).send({ version, recorded, device })
   } catch (err) {
     if (String(err).includes('duplicate')) {
       res.status(409).send({
-        error: 'already_registered',
-        error_description: 'A vehicle with this device_id is already registered'
+        error: new ConflictError('A vehicle with this device_id is already registered')
```

Comment:
Need to follow the same response format as the spec, see the [Agency Vehicles](https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/agency#vehicle---register) spec